### DISCUSSION TITLE
How to Deauthorize an API Key ?

### DISCUSSION THREAD

**Discussion by iis2h** [projectdiscovery/dnsx#603] [Q&A]
I've authenticated DNSX using an API key from https://cloud.projectdiscovery.io/, but the CLI tool no longer works. I think I need to pay for an API subscription. How can I remove the API key? Or should I uninstall and reinstall the tool? Also, the subscription fee is too expensive. Thanks.

**Comment by olearycrew** [comment]
@iis2h you shouldn't need to have a paid account in order for dnsx to work, the subscription is for cloud scanning on ProjectDiscovery Cloud Platform.

Can you tell me what error you're getting when trying to run dnsx? Can you also provide the output of `dnsx -hc`?

**Reply by olearycrew** [reply]
Also can you provide the output of `dnsx -auth` as that checks your authorization

**Comment by iis2h** [comment]
Thank you for your reply. I just figured it out, and the issue was in my settings. Thanks again.
